doc/examples/jupyter_notebook/Makefile

```diff
@@ -1,12 +1,11 @@
 build:
 	docker build -t pachyderm_jupyter .
 
-prep: build
+prep:
 	pachctl version
-	./citibike_data/load.sh
-	./weather_data/load.sh
+	cd citibike_data && ./load.sh
+	cd weather_data && ./load.sh
```

Comment:
Actually not, because it was uploading the directory structure to PFS as well (due to where the Makefile is located), and that was causing issues.